portfolio/src/main/webapp/script.js

```diff
@@ -26,3 +28,49 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomQuote() {
+    // TODO store quotes in seperate JSON file
+    const greetings =
+      ['Hello world!', '¡Hola Mundo!', '你好，世界！', 'Bonjour le monde!'];
+
+}
+
+function randomizeImage() {
+  // The images directory contains 11 images, so generate a random index between
+  // 1 and 11.
+  const size = 11;
+  const imageIndex = Math.floor(Math.random() * size) + 1;
+  const imgUrl = 'images/grace-' + imageIndex + '.jpg';
```

Comment:
Use the rich image format here:
`images/grace-${imageIndex}.jpg`;

Reply:
Doesn't seem to work for me when I do imgUrl = images/grace-${imageIndex}.jpg;
I get an error that { is an unexpected token, so the image doesn't load.

Reply:
Oops, github formatted my backticks:
\`images/grace-${imageIndex}.jpg\`;